packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,121 @@
+  - [Controlled Usage](#controlled-usage)
+- [Accessibility Considerations](#accessibility-considerations)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=52377%3A27686&mode=design&t=pFzCay0x4a6sTcoQ-1) to all variants of the `Chip` component.
+
+## `Chip` API
+
+> **Note**
+>
+> The `Chip` component is inherently tied to the `ChipGroup` and cannot be utilized outside its context.
+
+| Prop                    | Type                                                                 | Default      | Description                                                                                                                                        | Required |
+| ----------------------- | -------------------------------------------------------------------- | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| **children**            | `string`                                                             | `undefined`  | Renders the label text for the Chip                                                                                                                | ✅       |
+| **defaultChecked**      | `boolean`                                                            | `false`      | Sets the default value for an uncontrolled component                                                                                               | ❌       |
+| **icon**                | `Icon`                                                               | `undefined`  | Displays the Blade Icon component within the Chip                                                                                                  | ❌       |
+| **isChecked**           | `boolean`                                                            | `false`      | Controls the Chip's checked state                                                                                                                  | ❌       |
+| **isDisabled**          | `boolean`                                                            | `false`      | Disables or enables the Chip                                                                                                                       | ❌       |
+| **size**                | `'xsmall' \| 'small' \| 'medium' \| 'large'`                         | `'small'`    | Specifies the size of the rendered Chip                                                                                                            | ❌       |
+| **inputType**           | `'checkbox' \| 'radio'`                                              | `'checkbox'` | Determines the Chip's input type                                                                                                                   | ❌       |
```

Comment:
This probably doesn't need to be exposed to consumers, you can detect this internally since ChipGroup will have selectionType prop.